package.json

```diff
@@ -12,7 +12,7 @@
     "devDependencies": {
         "chai": "^4.1.2",
         "chai-as-promised": "^7.1.1",
-        "electron": "1.8.4",
+        "electron": "^1.8.4",
```

Comment:
Hello @iamonuwa Please don't change the electron dependency, I want it pinned to 1.8.4 for now. Also don't commit the package-lock.json for now.

Reply:
Alright, I'll revert it to the old version